frontend/src/main/web/src/components/GroupTeaser.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react' // eslint-disable-line
+import Link from '../components/Link'
+import View from '../components/View'
+import Icon from '../components/Icon'
+
+const viewTheme = {
+  base: {
+    ai: 'Ai(c)',
+    fld: '',
+    m: 'Mb(rh)'
+  }
+}
+
+const statusIcons = {
+  ACTIVE: '',
+  READONLY: 'locked',
+  OBSOLETE: 'trash'
+}
+
+const GroupTeaser = ({
+  details,
+  name,
+  children,
+  ...props
+}) => {
+  const status = statusIcons[details.status]
+  const description = details.description ? (
+      <div className={'C(muted)'}>
+        {details.description}
+      </div>
+    )
+    : (
+      <div className={'C(muted) Fs(i)'}>
+        No description available
+      </div>
+    )
+  const metaData = details.owner ? (
+    <View
+      theme={{
+        base: {
+          ai: 'Ai(c)',
+          fld: '',
+          fz: 'Fz(msn1)',
+          m: 'Mstart(a)--md'
+        }
+      }}>
+      <Icon
+        name='user'
+        size='n1'
+        theme={{ base: { c: 'C(muted)', m: 'Mend(rq)' } }}/>
+      <Link to={details.owner}>{details.owner}</Link>
+      <Icon
+        name='users'
+        size='n1'
+        theme={{
+          base: {
+            c: 'C(muted)',
+            m: 'Mend(rq) Mstart(rh)'
+          }
+        }}
+      />
+    </View>
+  ) : undefined
+  const useHref = true
+  const link = useHref
+    ? window.config.baseUrl + '/version-group/view/' + details.id
+    : 'groups/' + details.id
```

Comment:
`useHref` is always true. Either there is a mistake in setting `useHref`, or this can be simplified to a simple expression instead of a ternary expression.